lib/interactive.dart

```diff
@@ -222,7 +228,7 @@ class Interactive {
           ..setAttribute("y", y.toString())
           ..setAttribute("width", SQ_SIZE.toString())
           ..setAttribute("height", SQ_SIZE.toString())
-          ..setAttribute("fill", "black")
+          ..setAttribute("fill", _getThemeColor(colData[j].themes.first))
```

Comment:
This is actually a moderately significant decision that we're chose to use the first theme - so please could we extract it to a variable rather than doing it inline?

something like:

`displayTheme = colData[j].themes.first`

then
`..setAttribute("fill", _getThemeColor(displayTheme))`

Reply:
Fixed in the next PR